src/catalogue/service.go

```diff
@@ -220,12 +220,3 @@ func cut(products []Product, pageNum, pageSize int) []Product {
 	}
 	return products[start:end]
 }
-
-func contains(s []string, e string) bool {
-	for _, a := range s {
-		if a == e {
-			return true
-		}
```

Comment:
Where this change coming from?

Reply:
?

Reply:
[![wat](https://i.kym-cdn.com/photos/images/newsfeed/000/173/576/Wat8.jpg?1315930535)](https://i.giphy.com/yjs4noW0nqYc8.mp4)

Reply:
Just that I don’t have the context for why it’s being removed? Deprecated?

Reply:
Old Function. Not in use. Part of the "catalogue cleanup".

Reply:
👌